### PR TITLE
YouTube Clone -Review1

### PR DESCRIPTION
**Pull request by kimothokamau**
### YouTube Clone Project

This project mimics the YouTube video player page. It has the following main sections:

- A navigation bar
- A video player column
- A recommended videos column

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+This is the first project of the MIcroverse HTML/CSS Course.
+The requirement is to create a webpage that looks like the YouTube video player
+
+# Built With
+- HTML
+- CSS
+- Flexbox and Grid
+
+# Live Demo
+https://kimothokamau.github.io/youtubeclone_mv1/
```
Comment (ebeagusamuel) [inline README.md]: - [Required]
- [ ] Your readme is good but can still be improved on. You should add the screenshot of the cloned project. Here's a great [template ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)I strongly recommend you to use. You can select what sections to include, depending on your project.

File: assets/main.css [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+    list-style: none;
+    display: inline;
+}
+
+#search-bar > button {
+  padding: 0.5em;
+  border: 1px solid #d3d3d3;
+  background-color: #f0f0f0;
+}
+#search-bar > button:hover{
```
Comment (ebeagusamuel) [inline assets/main.css:84]: - [Optional]
- [ ] Minimize the use of id as CSS selectors. It is always best to use classes as selectors as id can not be reused but classes can.

**Review by ebeagusamuel — CHANGES_REQUESTED** [review]
### Changes Required :recycle: 

Hey team! Great work you've done on your first project. Super proud of you guys :+1: 
You guys met almost all the minimum requirements for the project, just some minor changes to be made.

Required changes :writing_hand: 
- There are some validation errors in your HTML. Please cross-check your code against an HTML validator and make necessary corrections.
- Update your readme as stated in the comment below.

#### Please make all required changes and submit your request again :smiley: 
If you have any questions, feel free to send me a message through Slack (@ebeagusamuel)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Ebeagu Samuel

**Review by ElijahTobs — APPROVED** [review]
# Status: Approved! :tada: 

Congratulations! @kimothokamau and @rebeccayilma 


### Good job implementing the change as requested by the previous TSE.

You can now merge and submit.

Happy Coding :computer: 

Feel free to reach out to me 
slack @ElijahTobs
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Pull request merged**